Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html

```diff
@@ -1,18 +1,34 @@
-<body>
+<body ng-app="">
   <div>
     <h1> Elysia Range Simulator </h1>
     <form name="simulator" #simulator="ngForm" novalidate>
       <label>Speed (km/h):</label>
       <br>
-      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)">
+      <input id="speed" name="speed" class="form-control" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)" required min="0" max="90" [formControl]="Sinput">
+      <br>
+      <label *ngIf="Sinput.invalid">
+        <label *ngIf="Sinput.errors.required" style="color:red">Speed is required.</label>
```

Comment:
When assigning the style for the error messages here add a space between the colon and 'red' and a semicolon after 'red' (i.e. color: red;)